otbenchmark/_MorrisSensitivity.py

```
@@ -260,17 +313,16 @@ def __init__(self, random_parameters=False):
         dimension = 20
         if random_parameters:
             b0 = ot.DistFunc.rNormal()
-            alpha = ot.DistFunc.rNormal(10)
-            beta = ot.DistFunc.rNormal(6 * 14)
-            gamma = ot.DistFunc.rNormal(20 * 14)
+            b1 = ot.DistFunc.rNormal(10)
+            b2 = ot.DistFunc.rNormal(175)
             warnings.warn(
```

Ruff (0.15.15)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)